Edits to tdigest and uddsketch (#1137)
timescale/docs: api/uddsketch.md

be.

Timescale's `UddSketch` implementation is provided as an aggregate function in
PostgreSQL. The output is currently only suitable as input to the percentile
approximation functions; it can be used directly as part of a one-off SQL query

> or as data stored in a continuous aggregate that will be queried later with

Reviewer: ```suggestion
or as data stored in a continuous aggregate that is queried later with
```

Author: applied!
